src/screens/SUE/SueReportScreen.tsx

```diff
@@ -279,10 +306,7 @@ export const SueReportScreen = ({
             return texts.sue.report.alerts.zipCode;
           }
 
-          if (
-            !!limitOfCity &&
-            limitOfCity.toLocaleLowerCase() !== getValues().city.toLocaleLowerCase()
-          ) {
+          if (!!limitOfCity && !areaServiceData?.postalCodes?.includes(getValues('zipCode'))) {
```

Comment:
can we ignore `limitOfCity` here, as it controls the query with `enabled`? so if `limitOfCity` is false, `areaServiceData?` isn't defined.

Reply:
Yes, that's right. If the `limitOfCity` value is not set, the query will not work. So the control here can be removed.